### PR TITLE
Manual npm publish

### PR DESCRIPTION
**Pull request by AdityaHegde**
Adding steps for manually publishing to npm

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -83,7 +87,7 @@
 		"d3-shape": "^3.1.0",
 		"d3-time": "^3.0.0",
 		"d3-time-format": "^4.1.0",
-		"duckdb": "^0.3.2",
+		"duckdb": "0.3.2",
```
Comment (AdityaHegde) [inline package.json:90]: duckdb 0.3.3 is throwing segmentation fault. hence fixing the version here.

**Comment by himadrisingh** [comment]
we should make these scripts windows compatible?

**Comment by AdityaHegde** [comment]
Would be overkill I think. In future we can have it similar to how our dash works. Once someone manually creates a tag we can run a job. These scripts will then be run on travis.

**Comment by hamilton** [comment]
@himadrisingh @AdityaHegde it would probably be good for us to consider doing this down the line for `package.json` scripts ~ see #168

File: package.json [hunk 2]
```diff
@@ -1,13 +1,13 @@
 {
-	"name": "rill-developer",
+	"name": "@rilldata/rill-developer",
```
Comment (hamilton) [inline package.json:2]: just want to flag that we should make sure this is what we want to go with.

**Review by hamilton — COMMENTED** [review]
This looks good. Finalizing is contingent on me getting clarity on the naming. Once we've established that, we can merge

File: package.json (hunk 2 above)
Comment (hamilton) [inline package.json:2]:
```suggestion
	"name": "@rilldata/rill",
```

File: package.json [hunk 3]
```diff
@@ -98,10 +101,11 @@
 		"svelte": "^3.46.4"
 	},
 	"bin": {
-		"data-modeler": "./dist/cli/data-modeler-cli.js"
+		"rill-developer": "./dist/cli/data-modeler-cli.js"
```
Comment (hamilton) [inline package.json:104]:
```suggestion
		"rill": "./dist/cli/data-modeler-cli.js"
```

**Review by hamilton — APPROVED** [review]
We decided to go with `@rilldata/rill` and `rill` for the bin. @AdityaHegde let met know if you need anything else. Once those changes are in, feel free to merge.

**Pull request merged**